auth-web/src/util/constants.ts

```diff
@@ -180,10 +181,13 @@ export enum CorpTypes {
     PARTNERSHIP = 'GP',
     SOLE_PROP = 'SP',
 
-    // colin
+    BC_CONTINUE_IN = 'C',
```

Comment:
Should we have this the same as our master list in the shared components library?

https://github.com/bcgov/bcrs-shared-components/blob/ab4c90c534948621f7cdcb7609c650f889f0f923/src/modules/corp-type-module/corp-type-module.ts#L14